modules/apps/commerce/commerce-service/src/main/resources/com/liferay/commerce/internal/batch/04-object-definition.batch-engine-data.json

```diff
@@ -128,13 +100,13 @@
 				{
 					"DBType": "String",
 					"businessType": "Aggregation",
-					"externalReferenceCode": "RETURN_CHANNEL_TOTAL_ACCEPTED",
+					"externalReferenceCode": "RETURN_CHANNEL_TOTAL_AMOUNT",
```

Comment:
why RETURN_CHANNEL_TOTAL_AMOUNT? I mean... why CHANNEL?

Reply:
they are names that if I remember correctly you actually chose them